tools/c7n_azure/c7n_azure/resources/network_security_group.py

```diff
@@ -92,6 +92,7 @@ class resource_type(ArmResourceManager.resource_type):
 EXCEPT_PORTS = 'exceptPorts'
 IP_PROTOCOL = 'ipProtocol'
 ACCESS = 'access'
+PREFIX = 'prefix'
```

Comment:
Can we update an existing test to test this property?

Reply:
Existing tests don't test the rule name at all, so short answer is no.

I filed an item to update tests\docs when we get some time. This specific issue was asked on gitter, so I think it worth spending 10 minutes for a quick fix now & do tests\docs later since it takes more time.